ifrs17-template/Test/IfrsVariablesTest.ipynb

```diff
@@ -28,7 +28,11 @@
         {
             "cell_type": "code",
             "source": [
-                "#!eval-notebook \"Data/InitSystemorphToMemoryForTesting\""
+                "#!import \"../Constants/CalculationEngine\"",
+                "\nvar constants = $\"#!import \\\"//{projectName}/{environmentName}/Constants/Consts\\\"\";",
+                "\n#!eval constants",
+                "\nProjectionCount = 1;",
```

Comment:
Do you still need this line?

Reply:
Yes, it is necessary because "#!eval constants" line doesn't assign a value to ProjectionCount.